### PR TITLE
code coverage increase iteration-5

### PR DESCRIPTION
**Pull request by lazerion**

### CODE REVIEW

File: hazelcast/src/test/java/com/hazelcast/mapreduce/impl/MapReduceUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        when(configuration.getNodeEngine()).thenReturn(nodeEngine);
+        when(configuration.getKeyValueSource()).thenReturn(new ListKeyValueSource());
+        when(nodeEngine.getClusterService().getSize(DATA_MEMBER_SELECTOR)).thenReturn(42);
+
+        JobProcessInformationImpl actual = MapReduceUtil.createJobProcessInformation(configuration, supervisor);
+        assertTrue(actual instanceof MemberAssigningJobProcessInformationImpl);
+
+        verify(nodeEngine, never()).getPartitionService();
+    }
+}
```
Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/mapreduce/impl/MapReduceUtilTest.java]: Missing newline at EOF

File: hazelcast/src/test/java/com/hazelcast/spi/impl/PortableCollectionTest.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        when(portableWriter.getRawDataOutput()).thenReturn(out);
+
+        PortableCollection collection = new PortableCollection(coll);
+        collection.writePortable(portableWriter);
+
+        verify(portableWriter).writeBoolean("l", true);
+        verify(portableWriter).writeInt("s", coll.size());
+        verify(out).writeData(data);
+    }
+}
```
Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/spi/impl/PortableCollectionTest.java]: Missing newline at EOF

File: hazelcast/src/test/java/com/hazelcast/mapreduce/impl/MapReduceUtilTest.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+import java.util.concurrent.TimeoutException;
+
+import static com.hazelcast.cluster.memberselector.MemberSelectors.DATA_MEMBER_SELECTOR;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.*;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class MapReduceUtilTest {
```
Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/mapreduce/impl/MapReduceUtilTest.java]: The whole MapReduce framework has been deprecated. The complete tests have been excluded in the `pom.xml`:
```xml
<exclude>**/mapreduce/**.java</exclude>
```
So I guess a new test here will not improve anything :)

File: hazelcast/src/test/java/com/hazelcast/mapreduce/impl/MapReduceUtilTest.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.util.concurrent.TimeoutException;
+
+import static com.hazelcast.cluster.memberselector.MemberSelectors.DATA_MEMBER_SELECTOR;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.*;
```
Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/mapreduce/impl/MapReduceUtilTest.java]: Unwanted star import

File: hazelcast/src/test/java/com/hazelcast/mapreduce/impl/MapReduceUtilTest.java (hunk 3 above)
Comment (lazerion) [reply hazelcast/src/test/java/com/hazelcast/mapreduce/impl/MapReduceUtilTest.java]: I see thanks, deleting added test.

**Comment by lazerion** [comment]
run-lab-run

File: hazelcast/src/test/java/com/hazelcast/spi/impl/PortableCollectionTest.java (hunk 2 above)
Comment (lazerion) [reply hazelcast/src/test/java/com/hazelcast/spi/impl/PortableCollectionTest.java]: Added

**Pull request merged**